### PR TITLE
Basic http test endpoint

### PR DESCRIPTION
**Pull request by dzaslavskiy**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+requests
```
Comment (folksgl) [inline requirements.txt]: Won't be using the standard python `requests` library with FastAPI as requests is a blocking http client. Instances of requests should be replaced with the more async-native `aiohttp` library.

File: manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+applications:
+  - name: usaccess
+    routes:
+      - route: ((ROUTE))
```
Comment (folksgl) [inline manifest.yaml]:
```suggestion
      - route: identity-give-usaccess-((ENVIRONMENT_NAME)).apps.internal
```

File: .github/workflows/deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          echo "::set-output name=ROUTE::$service_route"
+          echo "Deploying to $target_environment"
+
+      - name: Set CF CLI Target
+        run: cf target -o ${{ secrets.CF_ORG }} -s ${{ steps.deployment-target.outputs.SPACE }}
+
+      - name: Deploy application
+        run: cf push --vars-file vars.yaml
+          --var ROUTE=${{ steps.deployment-target.outputs.ROUTE }}
+          --strategy rolling
```
Comment (folksgl) [inline .github/workflows/deploy.yaml]:
```suggestion
              environment_name=dev
          elif [[ $GITHUB_REF =~ refs/tags/* ]]; then
              # Tags on main are deployed to test
              target_environment=give-test
              environment_name=test
          else
              echo 'Unknown deployment target'
              exit 1
          fi

          # Set the outputs for this step
          echo "::set-output name=SPACE::$target_environment"
          echo "::set-output name=ENVIRONMENT_NAME::$environment_name"
          echo "Deploying to $target_environment"

      - name: Set CF CLI Target
        run: cf target -o ${{ secrets.CF_ORG }} -s ${{ steps.deployment-target.outputs.SPACE }}

      - name: Deploy application
        run: cf push --vars-file vars.yaml
          --var ENVIRONMENT_NAME=${{ steps.deployment-target.outputs.ENVIRONMENT_NAME }}
          --strategy rolling
```

**Review by folksgl — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (dzaslavskiy) [inline requirements.txt:1]:
```suggestion

```

Comment (dzaslavskiy) [reply requirements.txt]: `requests` is used by `TestClient`

Comment (folksgl) [reply requirements.txt]: Ahh ok. In that case, `requests` should go into requirements-dev.txt, that way the application isn't pulling in extra libraries at runtime that aren't actual runtime dependencies.

**Review by folksgl — APPROVED** [review]

**Pull request merged**